Source/loadsave.cpp

```diff
@@ -363,18 +363,18 @@ static void LoadPlayer(LoadHelper *file, int p)
 	file->skip(3); // Alignment
 	pPlayer->_pSBkSpell = static_cast<spell_id>(file->nextLE<Sint32>());
 	pPlayer->_pSBkSplType = static_cast<spell_type>(file->nextLE<Sint8>());
-	for (int i = 0; i < 64; i++)
-		pPlayer->_pSplLvl[i] = file->nextLE<Sint8>();
+	for (int8_t &spellLevel : pPlayer->_pSplLvl)
+		spellLevel = file->nextLE<Sint8>();

```

Comment:
Why Sint8 if you changed stuff above to int8_t ? I'm confused :P

Reply:
`file->nextLE<Sint8>()` pull data out of the save data, it's not strictly related to the memory structures, you will see multiple places where we populate 32bit data type with data that we pulled out from an 8bit chunk. As such changing, one doesn't necessarily lead to the change other the other.

The use of SDL types will be generally cleaned up at a later point